UPGRADE-3.0.md

```diff
@@ -110,6 +110,9 @@ UPGRADE FROM 2.x to 3.0
 
 ### Form
 
+ * Type hints for `ChoiceList::initialize()` method's `$labels` and `$preferredChoices` 
+   parameters were removed.
```

Comment:
This should be documented in UPGRADE-2.8.md instead. Also, please add Before/After examples, like everywhere else in the UPGRADE files.